tests/app/broadcast_message/test_utils.py

```diff
@@ -87,12 +89,73 @@ def test_validate_and_update_broadcast_message_status_for_cancelling_broadcast(
     assert alert_event.message_type == BroadcastEventMessageType.CANCEL
 
 
-@pytest.mark.parametrize("reject_route", ["admin_interface", "api_call"])
-def test_validate_and_update_broadcast_message_status_for_rejecting_broadcast(
+def test_validate_and_update_broadcast_message_status_for_cancelling_broadcast_from_API_call(
     sample_broadcast_service,
     mocker,
-    reject_route
 ):
+    api_key = create_api_key(service=sample_broadcast_service)
+    template = create_template(sample_broadcast_service, BROADCAST_TYPE, content='emergency broadcast')
+    broadcast_message = create_broadcast_message(
+        template,
+        status=BroadcastStatusType.BROADCASTING,
+        areas={
+            "ids": ["london"],
+            "simple_polygons": [[[51.30, 0.7], [51.28, 0.8], [51.25, -0.7]]]
+        }
+    )
+    mock_task = mocker.patch('app.celery.broadcast_message_tasks.send_broadcast_event.apply_async')
+
+    validate_and_update_broadcast_message_status(
+        broadcast_message, BroadcastStatusType.CANCELLED, updating_user=None, api_key_id=api_key.id
+    )
+
+    assert broadcast_message.status == BroadcastStatusType.CANCELLED
+    assert broadcast_message.cancelled_at is not None
+    assert broadcast_message.cancelled_by_id is None
+    assert broadcast_message.cancelled_by_api_key_id == api_key.id
```

Comment:
@benthorner thanks for reviewing 🙏🏼 . I think we are testing that `cancelled_by_api_key_id` gets populated correctly here ☺️

Reply:
Indeed we are. I was talking about `**created**_by_api_key_id`. The old `api_key_id` column wasn't being checked before, though, so it's not a new issue. However, since we're making a change to the column, this PR is a good place to make sure there's a test to cover it somewhere.

Does that make sense?